src/scripts/delete-user.test.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable functional/no-return-void */
+/* eslint-disable functional/no-expression-statement */
+/* eslint-disable functional/functional-parameters */
+import * as TE from 'fp-ts/TaskEither';
+import * as O from 'fp-ts/Option';
+import { UserService } from './services/user-service';
+import { deleteUser } from './delete-user';
+
+import * as test from './__fixtures__/data-providers';
```

Comment:
Import baseUserService, etc explicitly.

Reply:
Should I implement the change for just delete-user.ts?

Reply:
Just this file. Keep PR small.

Reply:
Will do